### PR TITLE
TRUNK-5657 : Unable to display inactive conditions

### PR DESCRIPTION
**Pull request by varung-31**
## Description of what I changed

Correctly imported ConditionService.java and also removed code duplication

## Issue I worked on

see https://issues.openmrs.org/browse/TRUNK-5657

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

  No? -> [read here](https://wiki.openmrs.org/display/docs/Pull+Request+Tips) on how to squash multiple commits into one

- [x] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [x] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [x] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [x] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [x] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

**Comment by mozzy11** [comment]
Thanks @varung-31  for working on this. Can you please clean out all the unnecesary "white space" changes in your PR

**Comment by varung-31** [comment]
> Thanks @varung-31 for working on this. Can you please clean out all the unnecesary "white space" changes in your PR

I have not added any additional white spaces but only removed some of the unnecessary ones.
Do you still want me to revert ?

**Comment by mozzy11** [comment]
sure @varung-31 , better you revert them ,may be you could create another PR probably specifically for that. But for this PR , Only keep the changes required.

**Comment by varung-31** [comment]
Reopen

**Comment by varung-31** [comment]
> sure @varung-31 , better you revert them ,may be you could create another PR probably specifically for that. But for this PR , Only keep the changes required.

Made the changes!

**Comment by mozzy11** [comment]
great work @varung-31 .
 LGTM.
cc @dkayiwa

**Comment by mozzy11** [comment]
But your commit message should reflect what you have done ,not necessarily what was suggested in the  code review. Can you improve your commit message ??

**Comment by varung-31** [comment]
> But your commit message should reflect what you have done ,not necessarily what was suggested in the code review. Can you improve your commit message ??

Done.

File: api/src/main/java/org/openmrs/api/db/hibernate/HibernateConditionDAO.java [hunk 1]
```diff
@@ -15,6 +15,7 @@
 import org.hibernate.SessionFactory;
 import org.openmrs.Condition;
 import org.openmrs.Patient;
+import org.openmrs.api.ConditionService;
```
Comment (mozzy11) [inline api/src/main/java/org/openmrs/api/db/hibernate/HibernateConditionDAO.java]: do we need this import here ??

File: api/src/main/java/org/openmrs/api/db/ConditionDAO.java [hunk 2]
```diff
@@ -13,6 +13,7 @@
 
 import org.openmrs.Condition;
 import org.openmrs.Patient;
+import org.openmrs.api.ConditionService;
```
Comment (mozzy11) [inline api/src/main/java/org/openmrs/api/db/ConditionDAO.java]: do we need this import here ??

File: api/src/main/java/org/openmrs/api/db/hibernate/HibernateConditionDAO.java (hunk 1 above)
Comment (varung-31) [reply api/src/main/java/org/openmrs/api/db/hibernate/HibernateConditionDAO.java]: As you can see, there was a line in the original code : 
/**
* @see ConditionService#getAllConditions(Patient)
 */
To properly reference, ConditionService must be imported!

File: api/src/main/java/org/openmrs/api/db/ConditionDAO.java (hunk 2 above)
Comment (varung-31) [reply api/src/main/java/org/openmrs/api/db/ConditionDAO.java]: Same as the above!

**Comment by mozzy11** [comment]
ok , great

File: api/src/main/java/org/openmrs/api/db/ConditionDAO.java (hunk 2 above)
Comment (dkayiwa) [reply api/src/main/java/org/openmrs/api/db/ConditionDAO.java]: Do you get a compiler error when you remove the import?

File: api/src/main/java/org/openmrs/api/db/hibernate/HibernateConditionDAO.java (hunk 1 above)
Comment (dkayiwa) [reply api/src/main/java/org/openmrs/api/db/hibernate/HibernateConditionDAO.java]: Do you get a compiler error when you remove the import?

File: api/src/main/java/org/openmrs/api/db/ConditionDAO.java (hunk 2 above)
Comment (varung-31) [reply api/src/main/java/org/openmrs/api/db/ConditionDAO.java]: No, I don't get any compiler error. Do you want me to remove this ?

File: api/src/main/java/org/openmrs/api/db/hibernate/HibernateConditionDAO.java (hunk 1 above)
Comment (mozzy11) [reply api/src/main/java/org/openmrs/api/db/hibernate/HibernateConditionDAO.java]: @dkayiwa ,that import is only used by javadoc tags

Comment (dkayiwa) [reply api/src/main/java/org/openmrs/api/db/hibernate/HibernateConditionDAO.java]: Do we need it if we do not get a compiler error?

Comment (varung-31) [reply api/src/main/java/org/openmrs/api/db/hibernate/HibernateConditionDAO.java]: Removed the import.

**Comment by varung-31** [comment]
@mozzy11  This has been pending for a while. Could you please take a look ?

**Pull request merged**